tests/test_custom_admin.py

```diff
@@ -66,7 +66,7 @@ def test_custom_action_change_blog(self):
         data = {'action': 'change_blog',
                 '_selected_action': [f.pk for f in fixtures]}
         response = self.app.post(change_url, data, user=self.user)
-        form = response.form
+        form = response.forms['new_blog'] # http://webtest.pythonpaste.org/en/latest/api.html?highlight=multiple%20forms#webtest.response.TestResponse.form
```

Comment:
Django-suit added additional forms to the page, so `form = response.form` was raising a TypeError.  The [webtest docs](http://webtest.pythonpaste.org/en/latest/api.html?highlight=multiple%20forms#webtest.response.TestResponse.form) gave instructions on how to change this to select a specific form from multiple forms on the page.  I had to add an id attribute to the form tag in my template for this.